CleverRaven/Cataclysm-DDA
# Add "PALS_LARGE" to "single_sling"

#### Summary
None


#### Purpose of change

1. **Player choice**: Give the player the customization and choice of attaching the sling to their vest
2. **Realism**: We attach [single point slings to vests](https://camo.githubusercontent.com/44b777e1e994b9860d57a7d98e7c36ab767827aa04c4613ca95203c22503e30e/68747470733a2f2f7777772e746163746963616c6c696e6b2e636f6d2f6173736574732f696d616765732f70726f64756374696d616765732f696e666964656c736c696e672f73756c2e6a7067)
3. **Balance**: Currently the survival harness is by far the most efficient holder since most long guns can only be holstered to single point slings and survival harness. Single point slings can be used in conjunction with PALS holders but at the cost of much more encumbrance. This has the added benefit of freeing up "hang from front" strapped slot, while limited, adds to making pals/molle vests competitive.



#### Describe the solution
Allow single point slings to be attached to PALS/MOLLE capable gear such as load bearing vests and ballistic vests.

Chose the tag variant which takes up the most space(3) PALS_LARGE instead of medium or small for balance.


#### Describe alternatives you've considered
- Nerf survival harness by adding more encumbrance to be in line with single point slings used with PALS gear


#### Testing
- Check to see if it attaches (pass)
- Check to see if it can detach (pass)
- Check to see if a player can place a gun in the pocket (pass)


#### Additional context
![](https://i.imgur.com/cffnTit.png)
